### DISCUSSION TITLE
Exporting/backupping book positions

### DISCUSSION THREAD

**Discussion by snonux** [PaulWoitaschek/Voice#1740] [Q&A]
Hi,

Voice has been working great so far :-)

I am wondering whether there is a way to backup or export the bookmarks and the current audio book listening positions? 

Presumably, as a cheap solution, it could work just to backup a certain folder in Android/data?

Cheers!

**Comment by polyzen** [comment]
Duplicate of #1668

**Comment by astarq** [comment]
I'd like to add my vote to this. An export to .db, .backup, or .json like the vast majority of open source apps support would be so useful for those of us who don't want to or can't add a Google account to our devices and use Google's backup function!
